test/WebJobs.Script.Tests.Integration/TestScripts/Node/HttpTriggerReturn/index.js

```diff
@@ -0,0 +1,3 @@
+﻿module.exports = function (context, req) {
+    context.res.type("text/plain").send("test");
```

Comment:
The fluent apis call context.done internally (send, raw, json, sendStatus, end).

However, perhaps this isn't the right change. The problem is that all node http templates use $return by default, which I don't think should be the default pattern (as it doesnt save on code complexity and breaks this stuff in a weird way)